src/elements/content-sidebar/versions/VersionsSidebarAPI.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @flow
+ * @file Versions Sidebar API Helper
+ * @author Box
+ */
+import API from '../../../api';
+import { FILE_VERSION_FIELDS_TO_FETCH } from '../../../utils/fields';
+
+export type fetchPayload = [BoxItem, FileVersions];
+
+export default class VersionsSidebarAPI {
+    api: API;
+
+    fileId: string;
+
+    constructor({ api, fileId }: { api: API, fileId: string }) {
+        this.api = api;
+        this.fileId = fileId;
+    }
+
+    fetchData = (): Promise<fetchPayload> => {
+        return Promise.all([this.fetchFile(), this.fetchVersions()]).then(this.fetchVersionCurrent);
+    };
+
+    fetchDownloadUrl = (version: ?BoxItemVersion): Promise<string> => {
+        return new Promise((resolve, reject) => {
```

Comment:
Noticed below you're using `successCallback` and `errorCallback` in lieu of `resolve/reject`, should we make them consistent?

Reply:
Sure, I can update these to `successCallback` and `errorCallback`.